solutions/src/main/java/ru/sberbank/school/task08/Main.java

```diff
@@ -0,0 +1,34 @@
+package ru.sberbank.school.task08;
+
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+
+    private final static String url = "D:\\";
+    private final static String file = "def.txt";
```

Comment:
&#39;static&#39; modifier out of order with the JLS suggestions.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1236168380>*